examples/qtmegachatapi/MainWindow.cpp

```diff
@@ -422,10 +422,15 @@ void MainWindow::on_bSettings_clicked()
     auto actPrintMyInfo = menu.addAction(tr("Print my info"));
     connect(actPrintMyInfo, SIGNAL(triggered()), this, SLOT(onPrintMyInfo()));
 
+    menu.addSeparator();
+    auto actRetryPendingConn = menu.addAction(tr("Retry pending connections"));
+    connect(actRetryPendingConn, SIGNAL(triggered()), this, SLOT(onRetryPendingConnections()));
```

Comment:
We may want to add 2 options in the menu, rather than this one:
1. Retry pending connections
2. Force reconnect
The first one is the feature already added, the latter is passing a `true` to the `MegaChatApi` function, so it reconnects even if there's a connection already established.